auth0/resource_auth0_organization.go

```diff
@@ -74,83 +69,32 @@ func newOrganization() *schema.Resource {
 						"assign_membership_on_login": {
 							Type:     schema.TypeBool,
 							Optional: true,
+							Default:  false,
 						},
 					},
 				},
-				Set: hash.StringKey("connection_id"),
 			},
 		},
 	}
 }
 
 func createOrganization(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	organization := buildOrganization(d)
+	organization := expandOrganization(d)
```

Comment:
Renaming this so we're in line with the naming conventions "expand" and "flatten".